.buildkite/pipeline.perf-tests.yml

```diff
@@ -0,0 +1,30 @@
+# yaml-language-server: $schema=https://raw.githubusercontent.com/buildkite/pipeline-schema/main/schema.json
+
+env:
+  DOCKER_REGISTRY: "docker.elastic.co"
+  DOCKER_IMAGE: "${DOCKER_REGISTRY}/observability-ci/fleet-server" # needs to rename for rollback
+  DOCKER_IMAGE_GIT_TAG: "${BUILDKITE_BRANCH}" # needs to rename for rollback
+  DOCKER_IMAGE_LATEST_TAG: "latest" # needs to rename for rollback
+  DOCKER_IMAGE_SHA_TAG: "git-${BUILDKITE_COMMIT:0:12}" # needs to rename for rollback, should be "git-${BUILDKITE_COMMIT:0:12}"
+
+steps:
+  - label: ":docker: Publish docker image"
+    key: "create-image"
+    command: ".buildkite/scripts/build_push_docker_image.sh"
+    agents:
+      provider: "gcp"
+
+  - label: "perf test"
+    key: "obs-perf-test"
+    trigger: observability-perf
```

Comment:
my comment failed to trigger the job, is this the correct value I should be using, or should we use `fleet-smoke-tests` or `fleet-scale-tests` here?

Reply:
Changes in `.buildkite/pull-request.json` or `catalog-info.yaml` files are just applied when the PR is merged into main.

So, for the time being your comment would not trigger any build :( And your new pipeline would not exist neither in Buildkite.